core/worker.go

```diff
@@ -1704,7 +1698,7 @@ func (w *worker) processQiTx(tx *types.Transaction, env *environment, parent *ty
 		}
 		ETXPCount++ // conversion is technically a cross-prime ETX
 		if ETXPCount > env.etxPLimit {
-			w.logger.Error(fmt.Errorf("tx %032x emits too many cross-prime ETXs for block. emitted: %d, limit: %d", tx.Hash(), ETXPCount, env.etxPLimit))
+			//w.logger.Error(fmt.Errorf("tx %032x emits too many cross-prime ETXs for block. emitted: %d, limit: %d", tx.Hash(), ETXPCount, env.etxPLimit))
```

Comment:
This is a mistake sorry, pls deleteThis is a mistake sorry, pls delete

Reply:
deleted